.github/workflows/sdks.yml

```diff
@@ -161,7 +158,6 @@ jobs:
         if: ${{ (steps.cache-sdk.outputs.cache-hit != 'true') && (matrix.os == 'ubuntu-22.04') }}
         env:
           SWIFT_TAG: ${{ steps.version.outputs.tag }}
-          ANDROID_ARCH: ${{ matrix.arch }}
```

Comment:
This one is used by `get-packages-and-swift-source.swift` below, only the prior one isn't.